docs/en/stack/ml/anomaly-detection/ml-jobs-from-visuals.asciidoc

```diff
@@ -0,0 +1,63 @@
+[role="xpack"]
+[[ml-jobs-from-lens]]
+= Creating {anomaly-jobs} from Lens visualizations
+++++
+<titleabbrev>{anomaly-jobs-cap} from visualizations</titleabbrev>
+++++
+
+You can create {anomaly-jobs} from the compatible Lens charts on **Dashboard**. 
+
+[discrete]
+[[prereqs]]
+== Prerequisites
+
+* Only chart-like visualizations are compatible. Supported chart types are 
+`area`, `area_percentage_stacked`, `area_stacked`, `bar`, `bar_horizontal`, 
+`bar_horizontal_stacked`, `bar_percentage_stacked`, `bar_stacked`, and `line`.
+
+* The chart must contain a date field on one axis and it must be the same as the 
+default date field for the {data-source}.
+
+
+[discrete]
+[[create-job]]
+== Creating the job
+
+NOTE: You need to have a compatible visualization on **Dashboard** to create an 
+{anomaly-job}. If you don't have one but you want to try the feature out, go to 
+**Analytics > Dashboard** and select the `[Flight] Global Flight Dashboard` 
+which is based on the {kib} sample flight data set. Select the `Flight count` 
+visualization from the dashboard.
+
+
+. Go to **Analytics > Dashboard** and select a dashboard with a compatible 
+visualization.
+. Find the chart and select **More** from its **Options (...) menu**.
```

Comment:
I think this object is referred to as a "panel" (e.g. in https://www.elastic.co/guide/en/kibana/current/dashboard.html). If so, you could change to something like this:
```suggestion
. Open the **Options (...) menu** for the panel, then select **More**.  
```